docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
-| ------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Application Name                                                                                                                                       | Read-only field that shows the name of the application.                                                                                                |
-| HTTP Method                                                                                                                                            | Select the HTTP method to be used with update operations. Choices are:<br><ul><li><strong>Patch</strong> = patch requests are used to make changes to part of the resource at a location. This selection patches the resource, changing its properties. It is used to make minor updates to resources. </li><li><strong>Put</strong></strong> = puts resources at a location. Put requests always contain a full resource.</li>                                                                                  |
-| Application Specification                                                                                                                              | Select a local YAML or JSON file, enter a URL where the YAML or JSON file is located, or drag and drop the file to the location indicated in the form. |
+If you select **Redistribute** from the Quick Update drop-down list, one of two dialog boxes appears, depending on whether you elected to distribute your application per region or per PoP when you created it.
 
-4. Click **Submit & Deploy** to update your application automatically across ECP edge locations.<br>
+**Note:** Before proceeding, observe the following behaviors:<ul><li>The redistribution method cannot be changed using the Quick Update feature.</ul></li><ul><li>The application deployment process may take up to a few minutes, depending on the size of container images to be pulled, the target locations, and the number of expected instances. Once the application is deployed, **Deployed** appears as its status on the Applications page. To check the status and other details, use the applications list view and detail view. After the application is deployed, you can retrieve instances in [list view](</docs/portal/applications/retrieving-instances-of-an-application.md#retrieving-instances-of-an-application-in-list-view>) or [map view](</docs/portal/applications/retrieving-instances-of-an-application.md#retrieving-instances-of-an-application-in-map-view>)</ul></li><ul><li>Scaling down or redistributing an application that uses persistent storage volumes might delete existing persistent volumes along with pod instances.li></ul></li>
```

Comment:
There are unexpected characters at the end of the last list item, ".li>".

Reply:
Fixed.